.github/workflows/_test_pax_rosetta.yaml

```diff
@@ -20,138 +20,7 @@ on:
-          time srun \
-            --container-image=${{ steps.meta.outputs.IMAGE }} \
-            --container-mounts=${{ steps.meta.outputs.MODEL_PATH }}:/output \
-            --container-entrypoint \
-            test-pax.sh \
-              --output /output/${{ steps.meta.outputs.TEST_CASE_NAME }} \
-              --dtype bfloat16 \
-              --batch-per-gpu 4 \
-              --steps 500 \
-              --pipeline-parallel ${{ matrix.PARALLEL_CONFIG[0] }} \
-              --data-parallel ${{ matrix.PARALLEL_CONFIG[1] }} \
-              --fsdp ${{ matrix.PARALLEL_CONFIG[2] }} \
-              --tensor-parallel ${{ matrix.PARALLEL_CONFIG[3] }} \
-              --nodes ${{ steps.meta.outputs.NODES }} \
-              --enable-te \
-              ${{ inputs.EXTRA_TEST_ARGS }}
-          EOF
-          )
-
-          set +x
-          while sshx squeue -j $JOB | grep -q $JOB; do
-            echo "SLURM Job $JOB is still running."
-            sleep 15
-          done
-
-          echo "SLURM Job $JOB finished."
-
-          # Gather job info
-          SLURM_STATE=$(sshx sacct -j $JOB --format=State --parsable2 --noheader |& head -n 1)
-          SLURM_EXITCODE=$(sshx sacct -j $JOB --format=exitcode --parsable2 --noheader | sort -r -u | head -1 | cut -f 1 -d":" | sed 's/ //g')
-          echo "SLURM Job state is ${SLURM_STATE}"
-          echo "SLURM Job exit code is ${SLURM_EXITCODE}"
-          echo "SLURM_STATE=${SLURM_STATE}" >> "$GITHUB_OUTPUT"
-          echo "SLURM_EXITCODE=${SLURM_EXITCODE}" >> "$GITHUB_OUTPUT"
-
-          set -x
-
-      - name: Retrieve training logs and upload to TensorBoard server
-        shell: bash -x -e {0}
-        run: |
-          mkdir output/
-          rsync -rtz --progress \
-            ${{ secrets.CLUSTER_LOGIN_USER }}@${{ vars.HOSTNAME_SLURM_LOGIN }}:${{ steps.meta.outputs.LOG_FILE }} \
-            output/${{ steps.meta.outputs.TEST_CASE_NAME }}.log || true
-          rsync -rtz --progress \
-            ${{ secrets.CLUSTER_LOGIN_USER }}@${{ vars.HOSTNAME_SLURM_LOGIN }}:${{ steps.meta.outputs.MODEL_PATH }}/* \
-            output/ || true
-          rsync -rtz --progress \
-            output/ \
-            ${{ secrets.TENSORBOARD_UPLOAD_USER }}@${{ vars.HOSTNAME_TENSORBOARD }}:/tensorboard-logs/${GITHUB_RUN_ID}/ || true
-
-      - name: Write SLURM job status to file
-        shell: bash -x -e {0}
-        run: |
-          python << EOF
-          import json
-          with open("output/${{ steps.meta.outputs.TEST_CASE_NAME }}-status.json", "w") as f:
-              dump = {'state': "${{ steps.submit.outputs.SLURM_STATE }}", 'exitcode': "${{ steps.submit.outputs.SLURM_EXITCODE }}"}
-              json.dump(dump, f)
-          EOF
-        
-      - name: Upload training logs as artifacts
-        uses: actions/upload-artifact@v3
-        with:
-          name: ${{ steps.meta.outputs.JOB_NAME }}
-          path: output/*
-
```

Comment:
Same comment about the singleprocess tests

Reply:
Done